docker/Dockerfile-Ubuntu-16.04

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:16.04
```

Comment:
The docker images are not identical. 16.04 can't simply apt-get astyle so it has to download the .deb from the internet.

Reply:
Would the deb version working for 18.04 ? 

Maybe it would be worse... I don't know but I usually prefer to limit the amount of file we have to maintain....

Reply:
You could layer the dockerfiles and produce intermediate docker images
* base Ubuntu 16.04 & 18.04 through the astyle install 
* all the common steps in a dockerfile that takes a version argument

(At some point we may just want to stop testing on u16.04)

Reply:
Could also just put an `if` statement in the Dockerfile so if building with 18.04, it gets astyle as usual and if building with 16.04, it downloads the .deb

Reply:
This has been addressed. There is only 1 dockerfile now.